### PR TITLE
Travis PyPy coverage

### PR DESCRIPTION
**Pull request by reaperhulk**
Evolution of #2463 to see if this is an acceptable solution.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -46,17 +46,9 @@ basepython = python2.7
...
-[testenv:pypy]
-commands =
-    pip list
-    python -c "from cryptography.hazmat.backends.openssl.backend import backend; print(backend.openssl_version_text())"
-    py.test --capture=no --strict {posargs}
-
-# Temporarily disable coverage on pypy3 because of performance problems with
-# coverage.py on pypy3.
-[testenv:pypy3]
+[testenv:pypy-nocoverage]
```
Comment (alex) [inline tox.ini]: This needs `basepython = pypy`, otherwise it's just running CPython I assume :-)

**Comment by alex** [comment]
This looks like the right solution, emberassed I didn't htink of it.

File: tox.ini (hunk 1 above)
Comment (reaperhulk) [inline tox.ini]: Fixed!

**Comment by alex** [comment]
The bit in `upload_coverage` is broken, you can see if you expand the section in https://travis-ci.org/pyca/cryptography/jobs/100045029

**Comment by alex** [comment]
oops.

**Pull request merged**